dayrepo/snippets/urls.py

```diff
@@ -11,4 +11,5 @@
     path("employee/", views.get_employee,name="get_employee"),
     path("snippet-list/", views.snippet_list,name="snippet_list"),
```

Comment:
下の方でコメントしたが、パスにanippetが2回出てくるの違和感あるので以下でいかがか。
```suggestion
    path("/", views.snippet_list,name="snippet_list"),
```

こうすると、`localhost:xxxx/snippets/`で出てくるはず